adr/ADR-32.md

```diff
@@ -132,24 +132,137 @@ un-acked.
 ##### Options
 
 - `max_messages?: number` - max number of messages to return
-- `expires: number` - amount of time to wait for the request to expire
-  (required)
+- `expires: number` - amount of time to wait for the request to expire.
```

Comment:
specify unit of time.

Reply:
good call - plus, it's not actually required